CMakeLists.txt

```diff
@@ -317,7 +334,9 @@ if (MSVC)
 #$(SolutionDir)<toolset>msvc,<variant>release:<linkflags>/OPT:ICF=5
 #$(SolutionDir)<toolset>msvc,<variant>release:<linkflags>/OPT:REF
 elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-	add_definitions(-Wno-c++11-extensions)
+	if (abi STREQUAL "c++03")
```

Comment:
According to Wikipedia, C++03 is just a more polished wording of C++98, so features provided by both should be the same and Boost wouldn't change their definitions based on that. So I picked the newer one since it's better defined and certainly (hopefully?) old enough for any libtorrent users to have supported by their compiler.